src/transformers/models/chameleon/processing_chameleon.py

```diff
@@ -57,13 +80,9 @@ def __init__(self, image_processor, tokenizer, image_seq_length: int = 1024, ima
 
     def __call__(
         self,
-        text: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]] = None,
-        images: ImageInput = None,
-        padding: Union[bool, str, PaddingStrategy] = False,
-        truncation: Union[bool, str, TruncationStrategy] = None,
-        max_length: int = None,
-        return_tensors: Optional[Union[str, TensorType]] = TensorType.PYTORCH,
-        return_for_text_completion: bool = False,
+        text: Optional[Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]]] = None,
+        images: Optional[ImageInput] = None,
```

Comment:
I guess these two should be swapped

Reply:
I'm leaning on leaving these as-is as I'm not convinced there's any benefit in swapping them. Besides, the image-text-to-text pipeline could just not pass these as positional arguments. cc @yonigozlan

Reply:
I'll make a separate PR for swapping these args after this PR gets merged: https://github.com/huggingface/transformers/pull/32858#discussion_r1722516261